dss_datamover/tests/pytest_config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "config": "/etc/dss/datamover/config.json",
```

Comment:
Add socket related IP/Port in this file

Reply:
it is not needed, this is a pytest config file, not a datamover config file.